clients/common/norm.cpp

```diff
@@ -86,10 +85,10 @@ double norm_check_general<rocblas_half>(char norm_type,
     float alpha      = -1.0f;
     rocblas_int size = lda * N;
 
```

Comment:
With `host_vector<>` we don't have to use `.get()` or `.data().` to get the address. `host_vector<T>` automatically decays into `T*`